extensions/amp-list/0.1/amp-list.js

```diff
@@ -177,6 +170,12 @@ export class AmpList extends AMP.BaseElement {
     if (!this.element.getAttribute('src')) {
       return Promise.resolve();
     }
+    this.togglePlaceholder(true);
```

Comment:
This feature is technically is breaking UI change and I can see cases where one may not want this feature at all. I recommend putting this behind an attribute `show-loading-on-refresh` or similar. I will bring it up in our design review tomorrow morning to see what others think.

Reply:
Makes sense. Let me know the outcome of the design review and I'll update the code accordingly.

Reply:
Added to https://github.com/ampproject/amphtml/issues/13718#issuecomment-371205208.

tl;dr: `reset-on-refresh` attribute needed as opt-in signal to get this behaviour.